common/schemas.py

```diff
@@ -19,7 +19,7 @@
         t.StructField("titleType", t.StringType()),
         t.StructField("primaryTitle", t.StringType()),
         t.StructField("originalTitle", t.StringType()),
-        t.StructField("isAdult", t.BooleanType()),
+        t.StructField("isAdult", t.IntegerType()),
```

Comment:
I tried hard to compare bools, no use

Reply:
?

Reply:
I mean in filtering f.col("isAdult") == False, etc.